packages/vda-common/src/contract.ts

```diff
@@ -16,56 +18,60 @@ export interface CONTRACT_INFO {
     address: string
 }
 
-export const CONTRACT_ADDRESS : Record<CONTRACT_NAMES, Record<string, string | null>> = {
+const MAINNET = EnvironmentType.MAINNET
+const TESTNET = EnvironmentType.TESTNET
+const DEVNET = EnvironmentType.DEVNET
+
+export const CONTRACT_ADDRESS : Record<CONTRACT_NAMES, Record<EnvironmentType, string | null>> = {
```

Comment:
Not sure if it already exists, point me to it if so, but would it be helpful to have a kind of definition object for each network? Something like this?

```ts
export const myrtleDefinition: NetworkDefinition: {
  id: Network.MYRTLE,
  label: "Myrtle",
  isMainnet: true,
  anchoredBlockchain: BlockchainAnchor.POLPOS,
  storageNodeRegistryAddress: "0x...",
  // ... 
  // with the separation of DID and network,
  // add here only what's related to the network itself
}
```